client/cli/src/main.rs

```diff
@@ -509,12 +510,20 @@ async fn handle_command(
             }
         }
         Command::Info => {
+            let wallet = client.wallet_client();
+            let network: String = match wallet.config.network {
+                bitcoin::Network::Bitcoin => String::from("Bitcoin"),
+                bitcoin::Network::Testnet => String::from("Testnet"),
+                bitcoin::Network::Signet => String::from("Signet"),
+                bitcoin::Network::Regtest => String::from("Regtest"),
+            };
```

Comment:
[`Network` implements `Serialize`](https://docs.rs/bitcoin/0.29.2/bitcoin/network/constants/enum.Network.html#impl-Serialize-for-Network), so there is no reason to manually convert it to a string here.

Reply:
Fixed